test/internet.spec.ts

```diff
@@ -140,6 +140,29 @@ describe('internet', () => {
           expect(faker.definitions.internet.free_email).toContain(suffix);
         });
 
+        it('should not allow an email that starts with a .', () => {
+          const email = faker.internet.email('...Aiden');
+
+          expect(email).toBeTruthy();
+          expect(email).toBeTypeOf('string');
+          expect(email).toSatisfy(validator.isEmail);
+
+          const [prefix] = email.split('@');
+          expect(prefix).not.toMatch(/^(\.)/);
```

Comment:
Is there a reason why you put the dot in parenthesis?

```suggestion
          expect(prefix).not.toMatch(/^\./);
```